core/src/mercury/mod.rs

```
@@ -199,7 +199,7 @@ impl MercuryManager {
         if response.status_code >= 400 {
             warn!("error {} for uri {}", response.status_code, &response.uri);
             if let Some(cb) = pending.callback {
-                cb.complete(Err(MercuryError));
+                drop(cb.send(Err(MercuryError)));
```

Same thing about using `let _ = ` to ignore errors